omod/src/test/java/org/openmrs/module/fhir2/providers/r3/PersonFhirResourceProviderWebTest.java

```diff
@@ -408,4 +424,89 @@ private MockHttpServletResponse getPersonHistoryByIdRequest() throws IOException
 		return get("/Person/" + PERSON_UUID + "/_history").accept(FhirMediaTypes.JSON).go();
 	}
 	
+	@Test
+	public void createPerson_shouldCreatePerson() throws Exception {
+		String jsonPerson;
+		try (InputStream is = this.getClass().getClassLoader().getResourceAsStream(JSON_CREATE_PERSON_PATH)) {
+			Objects.requireNonNull(is);
+			jsonPerson = IOUtils.toString(is, StandardCharsets.UTF_8);
+		}
+
+		org.hl7.fhir.r4.model.Person person = new org.hl7.fhir.r4.model.Person();
+		person.setId(PERSON_UUID);
+
+		when(personService.create(any(org.hl7.fhir.r4.model.Person.class))).thenReturn(person);
+		
+		MockHttpServletResponse response = post("/Person").jsonContent(jsonPerson).accept(FhirMediaTypes.JSON).go();
+
+		assertThat(response, isCreated());
+	}
+
+	@Test
+	public void updatePerson_shouldUpdateExistingPerson() throws Exception {
+		String jsonPerson;
+		try (InputStream is = this.getClass().getClassLoader().getResourceAsStream(JSON_UPDATE_PERSON_PATH)) {
+			Objects.requireNonNull(is);
+			jsonPerson = IOUtils.toString(is, StandardCharsets.UTF_8);
+		}
+
+		org.hl7.fhir.r4.model.Person person = new org.hl7.fhir.r4.model.Person();
+		person.setId(PERSON_UUID);
+
+		when(personService.update(anyString(), any(org.hl7.fhir.r4.model.Person.class))).thenReturn(person);
+
+		MockHttpServletResponse response = put("/Person/" + PERSON_UUID).jsonContent(jsonPerson).accept(FhirMediaTypes.JSON)
+		        .go();
+
+		assertThat(response, isOk());
+	}
+
+	@Test
+	public void updatePerson_shouldErrorForNoId() throws Exception {
```

Comment:
Thanks @gitcliff  for this , i know this method does not affect the code however  by following code convention, would suggest something different like Public void updatePerson_shouldReturnErrorForNoId()  you can frase it your way , take a look at that method